code/datums/greyscale/json_configs/wellworn_shirt.json

```
@@ -0,0 +1,24 @@
+{
+	"wellworn_shirt": [
+		{
+			"type": "icon_state",
+			"icon_state": "worn_clean",
+			"blend_mode": "overlay",
+			"color_ids": [ 1 ]
+		}
+	],"wornout_shirt": [
```

If you have the time, these should probably be on separate lines like
```suggestion
	],
	"wornout_shirt": [
```
You did it to a few of 'em so you have to go through a few files to fix it tho